src/main/java/frc/robot/subsystems/swerve_module/SwerveModuleIoTalonFx.java

```diff
@@ -186,8 +186,7 @@ public void setTurnVoltage(double volts) {
 
     @Override
     public void updateInputs(SwerveModuleIoInputs inputs) {
-
-        BaseStatusSignal.refreshAll(
+        var result = BaseStatusSignal.refreshAll(
```

Comment:
Are you using the result variable? If so, strongly type it as per our code style; otherwise, don't save the return value at all.